test/WebSites/MinimalAppWithHostedServices/Program.cs

```diff
@@ -24,11 +24,21 @@ class HostedService : IHostedService
 {
     public Task StartAsync(CancellationToken cancellationToken)
     {
-        throw new Exception("Crash!");
```

Comment:
Was this here to test something specific?

Reply:
It was added intentionally (by me in turns out) as part of #2880 to verify that _other_ `IHostedService` aren't invoked by our CLI tool.

_Either_ that coverage needs to be retained somewhere else, or this project should be re-excluded from the Verification tests.

Reply:
Done. Added comment in Program.cs about this